Branch-TestBed-Swift/TestBed-Swift/Info.plist

```diff
@@ -102,5 +102,19 @@
 	<string>SDK not yet integrated</string>
 	<key>stitch_api_key</key>
 	<string>SDK not yet integrated</string>
+  <key>Fabric</key>
+  <dict>
+    <key>APIKey</key>
+    <string>a5689371c7d402061ef8dd47b44a89f0515ca4a7</string>
```

Comment:
Do we want this checked into github?

Reply:
This is a pretty low value key.

The key would allow people to submit crash reports on behalf of us.